### PR TITLE
MRG, FIX: add response_trig_shift parameter for BrainVision events

### PR DESCRIPTION
**Pull request by rkmaddox**
BrainVision .vmrk files (plaintext files where the events are stored) have both Stimulus and Response triggers, that use overlapping numbers. These are not treated differently by mne-python. Here is an excerpt of a .vmrk file:

```
Mk10=Stimulus,S  4,354587,1,0
Mk11=Stimulus,S  1,355015,1,0
Mk12=Response,R  1,1040956,1,0
Mk13=Response,R  1,1041049,1,0
```

These four triggers would be coded in the events array as 4, 1, 1, 1 respectively. With an Neuromag system, since the stimulus triggers and response triggers necessarily go into different bits, this problem is never encountered.

My proposed solution allows an optional `response_trig_shift` parameter that adds a constant shift to all response triggers. It defaults to 0 so as not to change current behavior (even though that behavior is arguably always erroneous), and a `None` value simply omits all response triggers. So with `response_trig_shift=100` the same four triggers would now be coded as 4, 1, 101, 101.

Does this seem like a reasonable fix? If so I will proceed with comments and tests.

### CODE REVIEW

File: mne/io/brainvision/brainvision.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ class RawBrainVision(_BaseRaw):
     @verbose
     def __init__(self, vhdr_fname, montage=None,
                  eog=('HEOGL', 'HEOGR', 'VEOGb'), misc=None, reference=None,
-                 scale=1., preload=False, verbose=None):
+                 scale=1., preload=False, verbose=None, response_trig_shift=0):
```
Comment (agramfort) [inline mne/io/brainvision/brainvision.py]: update docstring.

**Comment by agramfort** [comment]
travis is not happy. Also you can you add a super tiny test?

thanks @rkmaddox

File: mne/io/brainvision/brainvision.py (hunk 1 above)
Comment (Eric89GXL) [inline mne/io/brainvision/brainvision.py]: by convention `verbose` is always left as the last parameter, too

**Comment by rkmaddox** [comment]
Thanks @agramfort and @Eric89GXL. I think comments are addressed, waiting on Travis.

**Comment by rkmaddox** [comment]
Travis is happy.

**Comment by Eric89GXL** [comment]
LGTM, +1 for merge

Also, please change title prefix to `MRG` to make it clear it's ready for merge from your end (something we started doing fairly recently)

**Comment by rkmaddox** [comment]
Got it. Thanks much.

**Comment by agramfort** [comment]
thanks @rkmaddox

**Pull request merged**